docs/en/latest/installation-guide.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Installation
+keywords:
+- apisix-java-plugin-runner
+- Installation
+  description: This document explains how to installation and use apisix-java-plugin-runner.
```

Comment:
Hi, could you please add more descriptions to other docs? It's helpful on SEO :) (We will improve them of course, but need your help to provide the first version)

```suggestion
description: This document explains how to installation and use apisix-java-plugin-runner.
```

Reply:
fixed